ngraph/python/src/pyngraph/node.cpp

```diff
@@ -276,4 +276,10 @@ void regclass_pyngraph_Node(py::module m) {
         util::DictAttributeDeserializer dict_deserializer(attr_dict, variables);
         self->visit_attributes(dict_deserializer);
     });
+    node.def("_set_arguments", [](const std::shared_ptr<ngraph::Node>& self, const ngraph::OutputVector& arguments) {
+        return self->set_arguments(arguments);
+    });
+    node.def("_validate", [](const std::shared_ptr<ngraph::Node>& self) {
```

Comment:
I think this should be `validate` since it is going to be a public a public API from now on.

Reply:
Changed to 'validate'.